cognitive/src/main/scala/com/microsoft/azure/synapse/ml/services/openai/OpenAI.scala

```diff
@@ -324,6 +324,62 @@ trait HasOpenAITextParams extends HasOpenAISharedParams {
   }
 }
 
+trait HasRAIContentFilter {
+  /**
+   * Determines if the content in the output row was filtered by content safety
+   * @param outputRow The output row from the API response
+   * @return true if content was filtered, false otherwise
+   */
+  def isContentFiltered(outputRow: Row): Boolean
```

Comment:
make everything you can private[ml] thats not in the "public" api that you intend customers to be able to call

Reply:
theres also the protected and purely private modifiers as well, in general be a strict as you can